core/src/main/java/org/kiwix/kiwixmobile/core/settings/SettingsContract.kt

```diff
@@ -15,17 +15,13 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  *
  */
+package org.kiwix.kiwixmobile.core.settings
 
-package org.kiwix.kiwixmobile.core.settings;
+import org.kiwix.kiwixmobile.core.base.BaseContract
 
-import org.kiwix.kiwixmobile.core.base.BaseContract;
-
-interface SettingsContract {
-  interface View extends BaseContract.View<Presenter> {
-
-  }
-
-  interface Presenter extends BaseContract.Presenter<View> {
-    void clearHistory();
+internal interface SettingsContract {
```

Comment:
Why do you think `internal` will be appropriate?

Reply:
SHOULD I REMOVE THE INTERNAL ???

Reply:
Ok and also rebase your commits into one.

Reply:
sir my this branch has been corrupted with some unwanted commits and stuffs can i delete this pull request and create a fresh one ???

Reply:
You can close this and open a new one.